compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -1911,6 +1911,7 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
 
         def typeArgs: List[TypeRepr] = self match
           case AppliedType(_, args) => args
+          case AnnotatedType(parent, _) => parent.typeArgs
           case _ => List.empty
```

Comment:
Can you add a case for flexible types to better match the behaviour of `argInfos`?
```scala
case FlexibleType(underlying) => underlying.typeArgs
```

Reply:
I only ran BootstrappedCompilationTests that's why it was fast 😉 . I added the `FlexibleTypes` test,
'